pennylane/ops/qubit/observables.py

```diff
@@ -574,14 +574,14 @@ def label(self, decimals=None, base_label=None, cache=None):
         **Example:**
 
         >>> state_vector = np.array([0, 1, 1, 0])/np.sqrt(2)
-        >>> StateVectorProjector(state_vector, wires=(0, 1)).label()
+        >>> qml.Projector(state_vector, wires=(0, 1)).label()
```

Comment:
it can't be directly constructed because the `__new__` override doesn't take args. could fix it, but i'd like to only change docs as much as possible in this PR